model/sharing/open_note.go

```diff
@@ -34,22 +34,22 @@ func (n *apiNoteURL) Fetch(field string) []string            { return nil }
 // Opener can be used to find the parameters for creating the URL where the
 // note can be opened.
 type Opener struct {
-	*sharing.FileOpener
+	*FileOpener
 }
 
 // Open will return an Opener for the given file.
-func Open(inst *instance.Instance, fileID string) (*Opener, error) {
+func OpenNote(inst *instance.Instance, fileID string) (*Opener, error) {
```

Comment:
We also have an `Opener` for `model/office`. What do you think of moving it to `model/sharing` to have a more symmetrical approach?

Reply:
I was thinking about the same thing when I stumbled upon it while changing `Opener` into `NoteOpener` :).
I'll add a commit with this.